src/pyird/utils/aperture.py

```diff
@@ -29,11 +29,16 @@ def __init__(self, trace_function, y0, xmin, xmax, coeff):
 
     def mask(self):
         """mask image
-        
+
         Returns:
             mask image
 
         """
         return trace(self.trace_function, self.y0, self.xmin, self.xmax, self.coeff)
 
-    
+    def mmf2(self):
+        self.y0 = self.y0[::2]
+        self.xmin = self.xmin[::2]
+        self.xmax = self.xmax[::2]
+        self.coeff = self.coeff[::2]
+        return self
```

Comment:
To choose aperture only for star fiber (mmf2), I have added this function. I'm concerned that `return self` will not be a problem.

Reply:
you don't need `return self`

Reply:
Thank you! I fixed that part.